fast64_internal/sm64/sm64_geolayout_parser.py

```diff
@@ -574,16 +594,29 @@ def processBoneMeta(armatureObj, boneName, parentName):
     translateConstraint.target = armatureObj
     translateConstraint.subtarget = metaboneName
 
-    metabone.layers = createBoneLayerMask([boneLayers["meta"]])
+    if bpy.app.version >= (4, 0, 0):
+        if not "meta" in armature.collections:
+            armature.collections.new(name="meta")
+        armature.collections["meta"].assign(metabone)
+
+        if not "visual" in armature.collections:
+            armature.collections.new(name="visual")
+        armature.collections["visual"].assign(visualBone)
+
+        armature.collections["Ignore"].assign(visualBone)
```

Comment:
is the Ignore collection guaranteed to exist here? (idk)

Reply:
It is, but I'll add a check anyway.